zcash_client_backend/Cargo.toml

```diff
@@ -41,7 +45,8 @@ zcash_client_sqlite = { version = "0.2", path = "../zcash_client_sqlite" }
 zcash_proofs = { version = "0.4", path = "../zcash_proofs" }
 
 [features]
-test-dependencies = ["proptest", "zcash_primitives/test-dependencies"]
+transparent-inputs = ["ripemd160", "secp256k1"]
```

Comment:
looks like we either need another feature that turns on "hdwallet" or we need to add it to `transparent-inputs`.

Seems like a feature like "transparent-keys" might make sense because we mainly use hdwallet for it's `ExtendedPrivKey` and `KeyIndex`

Reply:
Alternatively, we can just have one feature to turn on everything we added here for transparent support. The trick would be to find the right name. `transparent-inputs` is actually not a bad overarching name because once you start adding transparent inputs, you will need all these things.